### DISCUSSION TITLE
how can i run a cron flow parallel ?

### DISCUSSION THREAD

**Discussion by alins1r** [PrefectHQ/prefect#5200] [Q&A]
this is my task
```
import datetime
import pendulum
import time

from prefect import Flow, task
from prefect.schedules import CronSchedule
from prefect.tasks.airtable import WriteAirtableRow
from prefect.tasks.github import GetRepoInfo
from prefect.triggers import any_failed


@task
def my_process():
    time.sleep(70)
    print("done")

daily_schedule = CronSchedule("*/1 * * * *")


with Flow("sleep more than cron", schedule=daily_schedule) as flow:
    my_process()

flow.run()
```
while i run this with `python3 flow.py` this run and **wait 70 second till end** and the task start again (_it takes two minutes run every task_) but i want it run **every minutes and don't care about the time the process** need to finished

1. run the flow
2. start the process
3. if its one minute after the running before started again
4. if 70 second of the first flow finished print done
5. and this process doing every minutes not every two minutes

**Comment by zanieb** [comment]
Hi. You'll need to use Prefect Cloud or Prefect Server to schedule your flow runs rather than running it locally like this.
